### DISCUSSION TITLE
How to make links between ADRs for showing in Decision Explorer (Structurizr Lite)?

### DISCUSSION THREAD

**Discussion by comdiv** [structurizr/lite#92] [Q&A]
1. Where to place links? (i've tried Status segment, Context segment)
2. What is a format exactly:
     - trys `(Some Text)[./0002-some-adr.md]` - not working at all (cannot navigate with MD, not seen in Explorer)
     - trys `(Some Text)[0002-some-adr.md]` - same
     - trys `(Some Text)[#2]` - starts work in MD itself (can navigate from one ADR to another), but Explorer still not sees links

**Comment by simonbrowndotje** [comment]
It's just a regular Markdown link; for example https://github.com/structurizr/examples/blob/main/java/src/main/java/com/structurizr/example/adrtools/decisions/0009-help-scripts.md is rendered as https://structurizr.com/share/39459/decisions/adr-tools#9

**Reply by comdiv** [reply]
It's not much clear that prefix before link is required (Amend in example)

Thx for answer

**Reply by JobaDiniz** [reply]
where do I add these adr files so they are picked up by the docker version?
